lib/activity/comments.js

```diff
@@ -2,10 +2,26 @@ const cache = require('../cache');
 const { Comment } = require('../messages/comment');
 
 module.exports = async (context, subscription, slack) => {
-  const { comment, repository } = context.payload;
+  const { repository } = context.payload;
 
   const issue = context.payload.issue || context.payload.pull_request;
 
+  let { comment } = context.payload;
+
+  // Fetch updated comment to get body_html
+  try {
+    comment = (await context.github.request({
+      method: 'GET',
+      url: context.payload.comment.url,
+      headers: { accept: 'application/vnd.github.html+json' },
+    })).data;
+  } catch (err) {
+    // Review comments API is not yet accessible by GitHub Apps
+    if (err.code !== 403) {
+      throw err;
+    }
```

Comment:
@wilhelmklopp what do you think about this? The review comments API is accessible by GitHub Apps yet, but everything else is. This allows mrkdwn rendering to work for all other comment types while we wait for that API to be enabled, and then we can remove this catch once it's done.

Reply:
Works for me! Could we maybe also include a logger.warn above the `throw err` that says something like 'Review comments not accessible to GitHub Apps', so that we can differentiate this error from all the other 403s we might get?

Reply:
Just to clarify, this will `catch` 403s and not report them. So the only other unexpected errors will get reported.

I had a `warn` before 5b2752c, but it causes a ton of unnecessary warnings to get reported in Sentry.